packages/teams-js/src/private/logs.ts

```diff
@@ -26,7 +26,7 @@ export namespace logs {
   export function registerGetLogHandler(handler: () => string): void {
     // allow for registration cleanup even when not finished initializing
     handler && ensureInitialized();
-    if (!isSupported()) {
+    if (handler && !isSupported()) {
```

Comment:
If `handler` is undefined for whatever reason, wouldn't this skip throwing that it is not supported?

Reply:
actually this should be fine

Reply:
Yes, it would! We've already been following that path in these functions (see line 28), specifically because we have some unit tests that don't call initialize (for whatever reason) and then part of cleaning up between tests is coming and in and calling functions like this with null handlers. I suspect there might be a better way to structure this long term, but outside the scope of this already huge PR